release/utils.py

```diff
@@ -55,12 +52,10 @@ def _is_commit_successful(commit_sha):
         Ensures there is at least one status update so that
         commits whose tests haven't started yet are not valid.
         """
-        statuses = github_api.commit_statuses(commit_sha)
-
-        # Determine if all statuses are success
-        passed_tests = all(status['state'] == 'success' for status in statuses)
+        commit_status = github_api.commit_statuses(commit_sha)
 
-        return len(statuses) == NUMBER_OF_TEST_SUITES and passed_tests
+        # Determine if the commit has passed all checks
+        return commit_status.get('state') == 'success'
```

Comment:
Following up on a conversation Akiva and I had later... He was talking about the case where travis/jenkins/etc has not acknowledged receipt of a commit. In other words: before the orange dot even appears. I looked at a couple of examples, and github stores those as 'pending', even if there is no CI hooked up at all. So we should be covered for that scenario.